snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```diff
@@ -105,6 +105,13 @@ metrics:
         column:
           OID: 1.3.6.1.4.1.9.9.13.1.3.1.6
           name: ciscoEnvMonTemperatureState
+        mapping:
+          1: normal
+          2: warning
+          3: critical
+          4: shutdown
```

Comment:
Isn't this a breaking change ? What if customers have monitors with `temp_state:3` ?

Reply:
No monitors are using this tag, only 5 dashboards, I think it's fine to keep this change